versioned_docs/version-v0.75/node-operators/get-started/setup-datanode.md

```diff
@@ -341,27 +343,56 @@ vega datanode init --home=$YOUR_DATANODE_HOME_PATH "vega-mainnet-0011"
 
 3. Change the data node configuration at `$YOUR_DATANODE_HOME_PATH/config/data-node/config.toml` to `AutoInitialiseFromNetworkHistory` = true
 
-4. Find the list of network history bootstrap nodes by querying the network history bootstrap API. For example: `https://api.vega.community/api/v2/networkhistory/bootstrap``
+4. Find the list of network history bootstrap nodes by querying the network history bootstrap API. For example: `https://api0.vega.community/api/v2/networkhistory/bootstrap``
 
-5. Still in your data node configuration file, paste the list of nodes into the section:
+5. Still in your data node configuration file, paste the list of nodes into the `NetworkHistory.Store.BootstrapPeers`:
 
 ```toml
   [NetworkHistory]
     [NetworkHistory.Store]
-      BootstrapPeers
+      BootstrapPeers = ["/dns/api1.vega.community/tcp/4001/ipfs/12D3KooWDZrusS1p2XyJDbCaWkVDCk2wJaKi6tNb4bjgSHo9yi5Q", "/dns/api2.vega.community/tcp/4001/ipfs/12D3KooWEH9pQd6P7RgNEpwbRyavWcwrAdiy9etivXqQZzd7Jkrh", "/dns/api3.vega.community/tcp/4001/ipfs/12D3KooWEH9pQd6P7RgNEpwbRyavWcwrAdiy9etivXqQZzd7Jkrh"]
+```
+
+6. Set number of blocks to sync to minimum value if you do not need historical data - it drastically improves startup time.
```

Comment:
```suggestion
6. Set number of blocks to sync to a low value if you do not need historical data - it drastically improves startup time.
```

Reply:
@daniel1302 I changed this because it implies to me that there's "a minimum" you should do. I think there is no set 'minimum', though, right? But feel free to reject this if I'm wrong